gremlin-python/src/main/python/gremlin_python/driver/protocol.py

```diff
@@ -19,6 +19,8 @@
 import logging
 import abc
 import base64
+import os
```

Comment:
is this import needed?

Reply:
oh prob missed from deleting the print outs